src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-traces.mdx

```diff
@@ -1,161 +1,122 @@
+
+<table>
+  <thead>
+    <tr>
+      <th>OTLP span proto field</th>
+      <th>New Relic `Span` field</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`ResourceSpans.Resource.attributes`</td>
+      <td>Each key/value is an attribute on the `Span` **[1]**</td>
+    </tr>
+    <tr>
+      <td>`ScopeSpans.InstrumentationScope.name`</td>
+      <td>`otel.library.name`</td>
+    </tr>
+    <tr>
+      <td>`ScopeSpans.InstrumentationScope.version`</td>
+      <td>`otel.library.version`</td>
+    </tr>
+    <tr>
+      <td>`ScopeSpans.InstrumentationScope.attributes`</td>
+      <td>Each key/value is an attribute on the `Span` **[1]**</td>
+    </tr>
+    <tr>
+      <td>`Span.trace_id`</td>
+      <td>`trace.id`</td>
+    </tr>
+    <tr>
+      <td>`Span.span_id`</td>
+      <td>`id`</td>
+    </tr>
+    <tr>
+      <td>`Span.trace_state`</td>
+      <td>`w3c.tracestate`</td>
+    </tr>
+    <tr>
+      <td>`Span.parent_span_id`</td>
+      <td>`parent.id`</td>
+    </tr>
+    <tr>
+      <td>`Span.name`</td>
+      <td>`name`</td>
+    </tr>
+    <tr>
+      <td>`Span.kind`</td>
+      <td>`span.kind`</td>
+    </tr>
+    <tr>
+      <td>`Span.start_time_unix_nano`</td>
+      <td>`timestamp`</td>
+    </tr>
+    <tr>
+      <td>`Span.end_time_unix_nano`</td>
+      <td>`druation.ms` (computed with `Span.start_time_unix_nano`</td>
+    </tr>
+    <tr>
+      <td>`Span.attributes`</td>
+      <td>Each key/value is an attribute on the `Span` **[1]**</td>
+    </tr>
+    <tr>
+      <td>`Span.dropped_attribute_count`</td>
+      <td>`otel.dropped_attributes_count`</td>
+    </tr>
+    <tr>
+      <td>`Span.events`</td>
+      <td>Each event is recorded as a `SpanEvent` with `span.id` / `trace.id` referring to source span, count stored as `nr.spanEventCount`</td>
+    </tr>
+    <tr>
+      <td>`Span.events[*].time_unix_nano`</td>
+      <td>Stored as `timestamp` on `SpanEvent`</td>
+    </tr>
+    <tr>
+      <td>`Span.events[*].name`</td>
+      <td>Stored as `name` on `SpanEvent`</td>
+    </tr>
+    <tr>
+      <td>`Span.events[*].attributes`</td>
+      <td>Each key/value is stored as an attribute on `SpanEvent`</td>
+    </tr>
+    <tr>
+      <td>`Span.events[*].dropped_attributes_count`</td>
+      <td>Stored as `ote.dropped_Attributes_count` on `SpanEvent`</td>
+    </tr>
+    <tr>
+      <td>`Span.dropped_events_count`</td>
+      <td>`otel.dropped_events_count`</td>
+    </tr>
+    <tr>
+      <td>`Span.status.message`</td>
+      <td>`otel.status_description`</td>
+    </tr>
+    <tr>
+      <td>`Span.status.code`</td>
+      <td>`otel.status_code`</td>
+    </tr>
+  </tbody>
+</table>
+
+**[1]** In case of conflict in resource attributes, scope attributes, span attributes, and top level span fields, the order of precedent (highest to lowest) is: top level `Span.*` fields > `Span.attributes` > `ScopeSpans.InstrumentationScope.attributes` > `ResourceSpans.Resource.attributes`.
```

Comment:
cc @ally-sassman for consistency with #17269
```suggestion

### Table footnotes [#otlp-mapping-notes]

**[1]** In case of conflict in resource attributes, scope attributes, span attributes, and top level span fields, the order of precedent (highest to lowest) is: top level `Span.*` fields > `Span.attributes` > `ScopeSpans.InstrumentationScope.attributes` > `ResourceSpans.Resource.attributes`.
```